.github/ISSUE_TEMPLATE/📲contribute-an-azd-template-.md

```diff
@@ -8,56 +9,32 @@ assignees: nitya, savannahostrowski
 ---
 
 ## New azd-template Submission
+Fill in the sections below to populate a new gallery card for your template. **All fields are mandatory unless explicitly identified as optional**
 
-Fill in the sections below to populate a new Gallery carry for your template. An example of a valid template object is given at the end for reference. **All fields are mandatory unless explicitly identified as optional**
+## 1. Description
+_1-2 sentence description of the architecture or solution that is defined by the template. If this is a requested template, please tag the GitHub issue containing the request._
 
-### 1. Title 
-_Short template name that will be used in card title on gallery_
+## 2. Author Information
+**Name:** _Name to credit on the gallery card_
 
-## 2. Description
-_1-2 sentence description of the architecture or solution that is defined by the template_
+**Link:** _To the author's GitHub profile (ideal) or other profile link_
 
-## 3. Author Profile
-_A link to the contributor's GitHub profile (ideal) or other profile link (for identification_.
+## 3. Template Repo
+_The GitHub repo containing the template being contributed-- must be a valid, complete template_
 
-## 4. Author Name
-_The name to be shown in the grey button on the card - will be linked profile website above_.
-
-## 5. Template Repo
-_The GitHub repo containing the template being contributed - must be a valid, complete template_
-
-## 6. Link to PR where you made your repo `azd` compatible
+## 4. Link to PR where you made your repo `azd` compatible
 _To allow us to provide feedback on your template, please include a link to the PR where you added `azd` compatibility_
 
-## 7. Standard Tags
-_One or more tags representing the template. Look at the currently available tags on the Gallery page and pick at least 1 tag for programming language used, and 1 tag for Azure services integrated_
-
-## 7. Additional Tags (`optional`)
-_Don't see a relevant standard tag for your template? Request these tags be added, and then used for your template. These could be new languages, services, or application domains_.
+## 5. Tags
+**Standard:** _One or more tags representing the template. Look at the currently available tags on the [Gallery page](https://azure.github.io/awesome-azd/) and pick **at least 1 tag** for programming language used, and **at least 1 tag for Azure services integrated**_
 
-## 8. Architecture Diagram or Application Screenshot
-_Used as default image for template card - must be representative of solution being enabled. Submit image by adding it as file upload to this issue_
+**Additional:** _Don't see a relevant tag for your template? Suggest tags to be added and used for your template. These could be new languages, services, or application domains_.
 
----
+## 6. Architecture Diagram or Application Screenshot
+_Used as display image for gallery card-- must be representative of solution being enabled._
```

Comment:
Do we have an guidance here? 

Perhaps what we mean is that the diagram should include all services + their connections etc.?